webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpParallelCoordinates.tsx

```diff
@@ -41,8 +41,8 @@ interface Props {
 }
 
 enum Action {
-  OpenTensorBoard = 'OpenTensorboard',
-  CompareTrials = 'CompareTrials'
+  OpenTensorBoard = 'View in TensorBoard',
+  CompareTrials = 'Compare Trials'
 }
```

Comment:
Since we're redefining the Action enum in all these files, should it be extracted to a types file?

Reply:
great idea 👍